api/db-cdr/generate-cdr/make-bq-data.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# This generates big query data that gets put in cloudsql
+# Counts and teh public and cdr data four cloudsql
+# note dev-up must be run to generate the schema
+# note run-local-data-migrations must be run to generate hard coded data from liquibase
+# note  the account must be authorized to perform gcloud and bq operations
+
+set -xeuo pipefail
+IFS=$'\n\t'
+
+
+# get options
+# --project=all-of-us-workbench-test *required
+
+# --cdr=cdr_version ... *optional
+
+USAGE="./generate-clousql-cdr/make-bq-data.sh --project <PROJECT> --account <ACCOUNT> --cdr-version=YYYYMMDD"
+while [ $# -gt 0 ]; do
+  echo "1 is $1"
+  case "$1" in
+    --account) ACCOUNT=$2; shift 2;;
+    --project) PROJECT=$2; shift 2;;
+    --cdr-version) CDR_VERSION=$2; shift 2;;
+    -- ) shift; break ;;
+    * ) break ;;
+  esac
+done
+
+if [ -z "${ACCOUNT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${PROJECT}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+if [ -z "${CDR_VERSION}" ]
+then
+  echo "Usage: $USAGE"
+  exit 1
+fi
+
+#Check cdr_version is of form YYYYMMDD
+if [[ $CDR_VERSION =~ ^[0-9]{4}(0[1-9]|1[0-2])(0[1-9]|[1-2][0-9]|3[0-1])$ ]]; then
+    echo "New CDR VERSION will be $CDR_VERSION"
+  else
+    echo "CDR Version doesn't match required format YYYYMMDD"
+    echo "Usage: $USAGE"
+    exit 1
+fi
+
+CREDS_ACCOUNT=${ACCOUNT}
+
+# Variables
+
+# TODO make this agument for cdr_vession
+bq_cdr_dataset="test_merge_dec26"
+NEW_BQ_CDR_DATASET=cloudsql_cdr$CDR_VERSION
```

Comment:
See above, we probably want "public" and "cdr" datasets here.... and putting "cloudsql_" in front of them seems weird... we intend researchers to be able to use these datasets directly, not just use them to generate CSV files to import into Cloud SQL. 

 Can we call these "cdr$CDR_VERSION" and "public$CDR_VERSION" or something like that?

Reply:
Public is no different then cdr except for a few rows deleted less than 12.  This can be refined.  Lets go with it for now and at end i will make public as a copy of cdr and delete what i need to from public.  Good ?